### ISSUE TITLE
Model Maker on Cluster

### ISSUE THREAD

**Issue by trudack** [ryanmcgreevy/ModelMaker#17]
Prepare a folder which can be copied to a cluster to run the Rosetta job there.

Take care of the renaming and best prepare the qsub submission script already

Make sure that the produced files can be read by model maker analysis

**Comment by ryanmcgreevy** [comment]
Note to self: 

1. update existing commands to write out cluster script but not run automatically, also using the updated directory structure and no hard-coded paths.
2. test mpi version

**Comment by ryanmcgreevy** [comment]
I have been able to get the MPI version of Rosetta working. I will commit an update tomorrow with an `-mpi <number of processors>` option for the modelmaker commands that will generate the appropriate scripts. It works pretty easily and just requires the mpi compiled version of Rosetta and calling the binary through `mpiexec -np<j>`.

**Comment by ryanmcgreevy** [comment]
[This commit](https://github.com/ryanmcgreevy/ModelMaker/commit/60600f11dc130870f8b58ca18c3505ed91131303) has support for using the MPI version of Rosetta, allowing for parallelization of `abinitio`, `refine`, and `insertion`. All the user has to do to use this, is set their `ROSETTAEXE` environment variable to `mpi.x` (where 'x' is the platform as usual) and include the `-np` option in any of the three commands, specifying the number of processors to use. 

I have tested all three commands on OSX. I know you guys are having problems getting the linux binaries to compile, which I will try to troubleshoot this afternoon, but the ModelMaker code should work on any platform so long as you have a successfully compiled MPI binary.

After that, I will look into getting the changes to the cluster functionality in, as @trudack and I discussed previously.
